### ISSUE TITLE
Introduce virtual scrolling

### ISSUE THREAD

**Issue by SamiKamal** [NeXTs/Clusterize.js#189]
Introduce virtual scrolling to overcome the height limit set by browsers

**Comment by robiningelbrecht** [comment]
Is there an actual height limit?

Virtual scrolling would be a great improvement though

**Comment by SamiKamal** [comment]
> Is there an actual height limit?
> 
> Virtual scrolling would be a great improvement though

Yes there's, it's different based on the browser, but on Chrome it's `512,000,000` pixels
